packages/core/src/external/commonwell/document/__tests__/document-downloader.test.ts

```diff
@@ -1,27 +1,36 @@
 import { APIMode, CommonWell, organizationQueryMeta } from "@metriport/commonwell-sdk";
 import * as stream from "stream";
 import { oid } from "../../../../domain/oid";
-import { getEnvVar } from "../../../../util/env-var";
+import { getEnvVarOrFail } from "../../../../util/env-var";
 import { S3Utils } from "../../../aws/s3";
 import { DocumentDownloaderLocal } from "../document-downloader-local";
 
-const cwOrgCertificate = getEnvVar("CW_ORG_CERTIFICATE");
-const cwOrgPrivateKey = getEnvVar("CW_ORG_PRIVATE_KEY");
-const bucketName = getEnvVar("MEDICAL_DOCUMENTS_BUCKET_NAME");
+const cwOrgCertificate = getEnvVarOrFail("CW_ORG_CERTIFICATE");
+const cwOrgPrivateKey = getEnvVarOrFail("CW_ORG_PRIVATE_KEY");
+const bucketName = getEnvVarOrFail("MEDICAL_DOCUMENTS_BUCKET_NAME");
+
+const region = getEnvVarOrFail("AWS_REGION");
+const fileName = getEnvVarOrFail("FILE_NAME");
+const fileLocation = getEnvVarOrFail("FILE_LOCATION");
+const mimeType = getEnvVarOrFail("MIME_TYPE");
+const fileSize = getEnvVarOrFail("FILE_SIZE");
+const size = fileSize ? parseInt(fileSize) : undefined;
+const orgName = getEnvVarOrFail("ORG_NAME");
+const orgOid = getEnvVarOrFail("ORG_OID");
+const orgNpi = getEnvVarOrFail("ORG_NPI");
```

Comment:
NABD, but I think we could avoid adding all these on the `.env.test` if we move these to within `describe.skip(`